docs/microphone.rst

```diff
@@ -0,0 +1,83 @@
+Microphone **V2**
+*****************
+
+.. py::module:: microbit
+
+This object lets you access the on-board microphone available on the **V2**
+micro:bit version. It can be used to respond to sound. The microphone input
```

Comment:
Should this be `micro:bit V2` instead of `V2 micro:bit version`?

Reply:
Updated in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5